src/components/sale/MemberPropertyModal.tsx

```diff
@@ -95,45 +95,44 @@ const MemberPropertyModal: React.FC<
     form
       .validateFields()
       .then(values => {
-        updateMemberProperty({
-          variables: {
-            memberId: member.id,
-            memberProperties: properties.map(property => {
-              const displayPropertiesName = displayProperties.map(displayProperty => displayProperty.name)
-              if (displayPropertiesName.includes(property.name)) {
-                return { member_id: member.id, property_id: property.id, value: values.property[property.id] || '' }
-              }
+        setLoading(true)
+        return Promise.allSettled([
```

Comment:
nice syntax of multiple asynchronous tasks

Reply:
nice syntax which provided by you